changes.xml

```diff
@@ -24,6 +24,9 @@
   <body>
 
     <release version="1.12.0" date="not released">
+      <action type="add" dev="cnagel">
+        CONGA Maven Plugin: Support 'conga.nodes' system property for selecting nodes to generate.
```

Comment:
it's not only the CONGA maven plugin, it affects whole CONGA (including CLI). and technically in this case it's a new maven property.